procedures/foreman/release.md.erb

```diff
@@ -62,23 +62,10 @@ Note: If for some reason there was an issue with the tarballs that required uplo
 
 # Packaging a release
 
-* [ ] In foreman-packaging rpm/<%= short_version %> branch, change foreman.spec, foreman-proxy.spec, foreman-selinux.spec, foreman-installer.spec:
-  * [ ] Set version to `<%= version %>`
-  * [ ] Set prerelease to `<%= extra %>`
-<% unless is_rc %>  * [ ] Reset release to `1`
-<% end %>  * [ ] In each package dir, remove the old tarball, run `spectool -g *.spec` and `git annex add *.tar.bz2`
-  * [ ] Commit with message "Release <%= full_version %>"
-  * [ ] Submit a pull request
-* [ ] Update changelog files for debs:
-  * [ ] `scripts/changelog.rb -v <%= debian_full_version %> -m "<%= full_version %> released" debian/*/*/changelog`
-  * [ ] Submit a pull request
+* [ ] Update [foreman-packaging](https://github.com/theforeman/foreman-packaging) rpm/<%= short_version %> and deb/<%= short_version %> branches
```

Comment:
What does update mean? Is there a more specific instruction?

Reply:
I am struggling with that a bit. I think I'll update https://projects.theforeman.org/projects/foreman/wiki/Release_Process and document the long form there that describes my script.

Reply:
Now that I've opened the repo: it's calling https://github.com/ekohl/theforeman-rel-eng/blob/master/bump_deb_packaging and https://github.com/ekohl/theforeman-rel-eng/blob/master/bump_rpm_packaging